src/analysis/special_functions/log/deriv.lean

```diff
@@ -268,4 +268,59 @@ begin
       by simpa [pow_succ'] using mul_le_of_le_one_right (pow_nonneg (abs_nonneg x) i) (le_of_lt h) }
 end
 
+/-- Power series expansion of `log(1 + x) - log(1 - x)` for `|x| < 1`. -/
+lemma has_sum_log_sub_log_of_abs_lt_1 {x : ℝ} (h : |x| < 1) :
+  has_sum (λ k : ℕ, (2 : ℝ) * (1 / (2 * k + 1)) * (x ^ (2 * k + 1))) (log (1 + x) - log(1 - x)) :=
+begin
+  have h₁, from has_sum_pow_div_log_of_abs_lt_1 h,
+  have h₂, from has_sum_pow_div_log_of_abs_lt_1 (eq.trans_lt (abs_neg x) h),
+  replace h₂ := (has_sum_mul_left_iff  (λ h : ( -1 = (0:ℝ)), one_ne_zero $ neg_eq_zero.mp h)).mp h₂,
+  rw [neg_one_mul, neg_neg, sub_neg_eq_add 1 x] at h₂,
+  have h₃, from has_sum.add h₂ h₁,
+  rw [tactic.ring.add_neg_eq_sub] at h₃,
+  let term := (λ n : ℕ, ((-1) * ((-x) ^ (n + 1) / ((n : ℝ) + 1)) + (x ^ (n + 1) / (n + 1)))),
+  let two_mul := (λ (n : ℕ), (2 * n)),
```

Comment:
Is there really a need for this auxiliary definition? Seems too simple to merit the trouble.

If there is, you could remove some parentheses:
```suggestion
  let two_mul := λ n : ℕ, 2 * n,
```